package.json

```diff
@@ -22,6 +22,9 @@
   "workspaces": [
     "modules/*"
   ],
+  "browser": {
+    "module-alias": false
```

Comment:
Why is this needed? Isn't module-alias a dev dependency?

Reply:
Yes it is. 

aliases is however now getting included in the browser-test build (to set the aliases for `readFile` which generates an ugly red error in the console log. I was working to remove all such noise from the log and this was one of the issues. 

Completely open to doing it in a different way... The call to moduleAlias should probably be moved out of `aliases.js` but was afraid of breaking something there.

Reply:
Move `addAlias` to `test/start.js` and dropped the `package.json` config.